game_eggs/steamcmd_servers/counter_strike/counter_strike_2/README.md

```diff
@@ -0,0 +1,44 @@
+# Counter-Strike 2
+
+Steam Description :
+For over two decades, Counter-Strike has offered an elite competitive experience, one shaped by millions of players from across the globe. And now the next chapter in the CS story is about to begin. This is Counter-Strike 2.
+
+## Install notes
+
+For CS2 Valve has decided to merge the game and server files. There will not be standalone dedicated server files.<br>
+
+## Installation/System Requirements
+
+|  | Minimum | Recommended |
+|---------|---------|---------|
+| Processor | minim x86-64-v2 (with popcnt) | --- |
+| RAM | 2 GB | 4 GB or better |
+| Storage | 33 GB | 50 GB (using Workshop Maps etc.) |
+| Packages | Glibc 2.31 | --- |
```

Comment:
This should not mather for docker aa that is image based

Reply:
[acb4bc6](https://github.com/parkervcp/eggs/pull/2725/commits/acb4bc61673feac4520df7043218729b971c9ec7)